config/locales/in_person_proofing/en.yml

```diff
@@ -44,7 +44,7 @@ en:
         none_found: No locations found.
```

Comment:
General design feedback: Changes look good to me. Thanks for sending a screenshot 👍